pkg/preflight/analyze.go

```diff
@@ -77,7 +78,11 @@ func doAnalyze(allCollectedData map[string][]byte, analyzers []*troubleshootv1be
 	for _, analyzer := range analyzers {
 		analyzeResult, err := analyze.Analyze(analyzer, getCollectedFileContents, getChildCollectedFileContents)
 		if err != nil {
-			strict, _ := HasStrictAnalyzer(analyzer)
+			strict, err := HasStrictAnalyzer(analyzer)
```

Comment:
does this overwrite the error on 79 if nil?

Reply:
does this overwrite the error on 79 if nil?

Reply:
@emosbaugh `Yes, it will.
but, now that I think of we should preserve analyzer error and return it as part of analyzeResult.
Updating the err checks.